src/main/kotlin/it/pagopa/checkout/authservice/mdcutilities/AuthenticationTracingUtils.kt

```diff
@@ -0,0 +1,95 @@
+package it.pagopa.checkout.authservice.mdcutilities
+
+import reactor.util.context.Context
+
+/**
+ * Tracing utility class that contains helper methods to set authentication information, such as
+ * client IP and RPT ID into reactor context
+ */
+class AuthenticationTracingUtils {
+
+    /**
+     * Tracing keys enumerations that contains both context key and default value, set in case such
+     * information are not taken from incoming request
+     */
+    enum class TracingEntry(val key: String, val defaultValue: String) {
+        RPT_ID("rptId", "{rptId-not-found}"),
+        CLIENT_IP("clientIp", "{client-ip-not-found}"),
+        API_ID("apiId", "{api-id-not-found}"),
+    }
+
+    /** Authentication information record */
+    // in Kotlin the data class provides equals(), toString() etc.
+    data class AuthenticationInfo(
```

Comment:
```suggestion
    data class RequestTracingInfo(
```

🚲 what about refactor this data class (and other tracing classes) to a more generic RequestTracingBlaBlaBla?  here AuthenticationInfo seems to be an authentication related feature and can cause naming clashes with domain objects

Reply:
Good point, addressed with [f67e3c3](https://github.com/pagopa/pagopa-checkout-auth-service/pull/14/commits/f67e3c33a82fb357324e8c9bac395fd7d79fe213) ✅